### PR TITLE
Try to quickly fix the verifiers

### PR DESCRIPTION
**Pull request by david-a-wheeler**

### CODE REVIEW

File: .github/workflows/verifiers.yml [hunk 1]
```diff
@@ -144,7 +144,7 @@ jobs:
       - run: python3 scripts/latex_collector.py set.mm list-set.tex
       - run: pdflatex -halt-on-error list-set.tex 2>&1 | tee ,latex-output
       # Complain if the pdflatex output says something is invalid
-      - run: ! grep ' invalid ' ,latex-output
+      - run: "! grep ' invalid ' ,latex-output"
```
Comment (wlammen) [inline .github/workflows/verifiers.yml:147]: Shouldn't it be `- run: ! grep " invalid " latex-output` instead?  Following usual command line syntax and the pattern in line 142

Comment (jkingdon) [reply .github/workflows/verifiers.yml:147]: Once we have the build passing, I'm all for spending more time with https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell or whatever will help us gain full understanding.

I'm going to merge this now, however. Feel free to open an issue or new pull request or any other means, I just think we need to do either this or #3134 while we have that discussion.

**Review by jkingdon — APPROVED** [review]

**Comment by wlammen** [comment]
After triggering the CI script on #3130 the metamath-exe check now chokes on this line
- run: sudo apt-get install texlive-latex-extra texlive-extra-utils texlive-fonts-extra texlive-science
Took 7 min 19 sec to complete.
Hopefully a one-time problem that gets better once caches are filled.

**Comment by jkingdon** [comment]
> After triggering the CI script on #3130 the metamath-exe check now chokes on this line
> 
>     * run: sudo apt-get install texlive-latex-extra texlive-extra-utils texlive-fonts-extra texlive-science
>       Took 7 min 19 sec to complete.
>       Hopefully a one-time problem that gets better once caches are filled.

Apparently so because I'm now seeing "All checks have passed" there.

**Pull request merged**